### DISCUSSION TITLE
Add utility for `text-wrap: balance` (`text-balance`)

### DISCUSSION THREAD

**Discussion by PositiveUser** [tailwindlabs/tailwindcss#11486] [Ideas]
[A video on `text-wrap: balance` by Kevin Powell](https://youtu.be/snKm-VzJCOo)

**Comment by wongjn** [comment]
Already include via #11320 though unreleased as of yet. You can test via the insiders build

```sh
npm install tailwindcss@insiders
```

**Reply by Norfeldt** [reply]
It's just that it was showed off as a feature in tailwind connect https://youtu.be/CLkxRnRQtDE?t=1058 (didn't sound like an insider build)

**Reply by wongjn** [reply]
At [18:11](https://youtu.be/CLkxRnRQtDE?t=1091), Sam Selikoff said after demonstrating `text-balance`:

> This is a new utility coming to the next version of Tailwind […]

The "next version of" phrase I feel would imply that it is not yet released.

**Reply by Norfeldt** [reply]
I must have missed that 🙉 - thx for point that out.
